initfiles/componentfiles/configxml/ldapserver.xsd

```diff
@@ -162,6 +162,14 @@
                     </xs:appinfo>
                 </xs:annotation>
             </xs:attribute>
+            <xs:attribute name="ldapTimeout" type="xs:nonNegativeInteger" use="optional" default="60">
```

Comment:
I like to be explicit on the timeout metric, in this case it might be helpful to name this attribute ldapTimeoutSecs (even though you did explain below)